--- origsrc/gavl-1.1.2/gavl/timer.c	2010-01-14 12:32:51.000000000 -0600
+++ src/gavl-1.1.2/gavl/timer.c	2010-10-16 22:34:39.298981700 -0500
@@ -30,7 +30,7 @@
 #endif
 #include <sys/time.h>
 
-#include <gavltime.h>
+#include <gavl.h>
 
 #ifdef HAVE_CLOCK_MONOTONIC
 
@@ -141,7 +141,7 @@ uint64_t gavl_benchmark_get_time(int con
 #endif
   }
 
-const char * gavl_benchmark_get_desc(int flags)
+const char * gavl_benchmark_get_desc(int config_flags)
   {
 #ifdef ARCH_X86
   uint64_t x;
@@ -153,7 +153,7 @@ const char * gavl_benchmark_get_desc(int
   else
     {
 #endif
-    return "microseconds returned by gettimeofday"
+    return "microseconds returned by gettimeofday";
 #ifdef ARCH_X86
     }
 #endif
