--- origsrc/gavl-1.1.2/src/blend_test.c	2010-01-14 12:32:51.000000000 -0600
+++ src/gavl-1.1.2/src/blend_test.c	2010-10-16 22:41:23.384912000 -0500
@@ -179,7 +179,7 @@ static gavl_video_frame_t * read_png(con
     {
     case PNG_COLOR_TYPE_GRAY:       /*  (bit depths 1, 2, 4, 8, 16) */
       if(bit_depth < 8)
-        png_set_gray_1_2_4_to_8(png_ptr);
+        png_set_expand_gray_1_2_4_to_8(png_ptr);
       if (png_get_valid(png_ptr, info_ptr, PNG_INFO_tRNS))
         {
         png_set_tRNS_to_alpha(png_ptr);
--- origsrc/gavl-1.1.2/src/pngutil.c	2010-01-14 12:32:51.000000000 -0600
+++ src/gavl-1.1.2/src/pngutil.c	2010-10-16 22:40:53.389955400 -0500
@@ -173,7 +173,7 @@ gavl_video_frame_t * read_png(const char
     {
     case PNG_COLOR_TYPE_GRAY:       /*  (bit depths 1, 2, 4, 8, 16) */
       if(bit_depth < 8)
-        png_set_gray_1_2_4_to_8(png_ptr);
+        png_set_expand_gray_1_2_4_to_8(png_ptr);
       if (png_get_valid(png_ptr, info_ptr, PNG_INFO_tRNS))
         {
         png_set_tRNS_to_alpha(png_ptr);
--- origsrc/gavl-1.1.2/src/scaletest.c	2010-01-14 12:32:51.000000000 -0600
+++ src/gavl-1.1.2/src/scaletest.c	2010-10-16 22:41:13.010893900 -0500
@@ -210,7 +210,7 @@ static gavl_video_frame_t * read_png(con
     {
     case PNG_COLOR_TYPE_GRAY:       /*  (bit depths 1, 2, 4, 8, 16) */
       if(bit_depth < 8)
-        png_set_gray_1_2_4_to_8(png_ptr);
+        png_set_expand_gray_1_2_4_to_8(png_ptr);
       if (png_get_valid(png_ptr, info_ptr, PNG_INFO_tRNS))
         {
         png_set_tRNS_to_alpha(png_ptr);
