--- origsrc/gavl-1.1.0/gavl/Makefile.am	2008-08-20 15:18:08.000000000 -0500
+++ src/gavl-1.1.0/gavl/Makefile.am	2009-03-04 16:45:24.382875000 -0600
@@ -93,7 +93,7 @@ csp_packed_planar.h  \
 csp_planar_packed.h  \
 csp_planar_planar.h
 
-AM_LDFLAGS = -export-symbols-regex gavl -version-info @LTVERSION_CURRENT@:@LTVERSION_REVISION@:@LTVERSION_AGE@
+AM_LDFLAGS = -export-symbols-regex gavl -version-info @LTVERSION_CURRENT@:@LTVERSION_REVISION@:@LTVERSION_AGE@ -no-undefined
 
 libgavl_la_LIBADD = \
 @GMERLIN_DEP_LIBS@ \
