Remove for cygwin-1.7.

--- origsrc/gavl-1.1.0/gavl/c/_sampleformat_c.c	2008-01-15 18:17:41.000000000 -0600
+++ src/gavl-1.1.0/gavl/c/_sampleformat_c.c	2009-03-04 17:00:33.726625000 -0600
@@ -19,6 +19,11 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * *****************************************************************/
 
+#ifdef __CYGWIN__ /* <= 1.7.0 */
+#define llrint rint
+#define llrintf rintf
+#endif
+
 static void RENAME(swap_sign_8)(gavl_audio_convert_context_t * ctx)
   {
   CONVERSION_FUNC_START
